core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -34,6 +35,7 @@
             </a>
         </ng-container>
         <ng-container right>
+            <p *ngIf="!es.editor.page.canEdit">You don't have permission to edit this page.</p>
```

Comment:
Update to the new design.

Reply:
https://www.figma.com/file/trbwjLuf5F6KaU1rFucNeg/Edit-Page?type=design&node-id=2623%3A13355&mode=design&t=Rs6SCq55j2x09xl4-1